sdk/src/main/java/com/hedera/hashgraph/sdk/MirrorNodeRouter.java

```diff
@@ -0,0 +1,69 @@
+/*-
+ *
+ * Hedera Java SDK
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.hedera.hashgraph.sdk;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+class MirrorNodeRouter {
+
+    private static final String API_VERSION = "/api/v1";
+
+    // refactor me
+    static String LOCAL_NODE_PORT = "5551";
```

Comment:
Can this be made configurable via some config file or something like this?

Reply:
@0xivanov Unfortunately, no, I mentioned it under "Other points" in the PR description.